Formula/nbdime.rb

```diff
@@ -20,53 +21,42 @@ class Nbdime < Formula
   end
 
   depends_on "ipython"
-  depends_on "jsonschema"
```

Comment:
Wasn't even used since this is indirect dependency via `jupyterlab` dependency tree.